sunpy/cm/tests/test_cm.py

```diff
@@ -0,0 +1,16 @@
+import sunpy.cm
+
+import pytest
+import sunpy.cm as cm
+import matplotlib.pyplot as plt
+
```

Comment:
Import for sunpy.cm is repeated. 

I think the order should be 

```
import pytest
import matplotlib.pyplot as plt

import sunpy.cm as cm
```

Reply:
Fixed it.